lib/shared/pe_status_check.rb

```diff
@@ -146,6 +150,28 @@ def self.filesystem_free(path)
     (stat.blocks_available.to_f / stat.blocks.to_f * 100).to_i
   end
 
+  #Get the maximum and current concurrent connections to Postgres
+  def self.psql_return_result(sql, psql_options = '')
+    command = %(su pe-postgres --shell /bin/bash --command "cd /tmp && #{PUP_PATHS[:server_bin]}/psql #{psql_options} --command \\"#{sql}\\"")
+    Facter::Core::Execution.execute(command)
+  end
+
+  def self.max_connections
+    sql = %(
+    SELECT current_setting('max_connections');
+  )
+    psql_options = '-qtAX'
```

Comment:
Maybe make this the default `psql_options` on line 154 so you don't have to pass it every time you call the method.

```
def self.psql_return_result(sql, psql_options = '-qtAX')
```